pkg/network/setup/network.go

```diff
@@ -35,22 +35,20 @@ type VMNetworkConfigurator struct {
 	vmi          *v1.VirtualMachineInstance
 	handler      netdriver.NetworkHandler
 	cacheCreator cacheCreator
+	launcherPid  *int
```

Comment:
Can this be a non-pointer?

Reply:
It has to be a pointer since it can be `nil`. `VMNetworkConfigurator` is used in `phase2` by virt-launcher and there the `launcherPid` is not relevant (you're inside the virt-launcher), so it has to be `nil`. `podNic` uses the same convention.